docs/general/foundation/testnet/README.md

```diff
@@ -45,6 +48,8 @@ The Oasis Node is part of the Oasis Core release.
 
 :::
 
+[handling network upgrades]: ../../run-a-node/maintenance-guides/handling-network-upgrades.md
```

Comment:
Hm the link in question doesn't seem to answer the question on what to do when syncing from genesis. There should be a special section on that page called "Syncing from genesis" explicitly explaining what to do in this case (e.g. when there have been _multiple seamless upgrades_ since genesis) even if it seems obvious.

Use an example where two (or three) versions are specified in network parameters and explain the steps where the earliest version is downloaded first and syncing is started using that one until the node stops for upgrade by itself. Then continue with the next version until the same happens and finally use the latest specified version.

Reply:
Revamped the beginning so that it also puts syncing from genesis into context.